src/site/_includes/components/pageheader.njk

```diff
@@ -1,5 +1,42 @@
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 <script src="https://unpkg.com/mermaid/dist/mermaid.min.js"></script>
+<script src="https://unpkg.com/lucide@0"></script>
+<script>
+    // Create callout icons
+    window.addEventListener("load", () => {
```

Comment:
This is the questionable JS

Reply:
I don't think this is questionable at all. It's clean, and gets the job done.